src/include/iscsi-lio-server/helps.rb

```diff
@@ -129,10 +131,10 @@ def initialize_iscsi_lio_server_helps(include_target)
          ) +
           _(
             "<p>With <b>Edit LUN</b> one can modify the LUN mapping. Please note that LUN target number" +
-            " must be unique.<br>After pressing <b>Edit Auth</b>, select the type of authentication." +
-            " Use <b>Incoming</b>, <b>Outgoing</b> or both together. Then insert <b>User</b> and <b>Password</b>." +
-            " Please make sure to set different passwords for incoming and outgoing authentication.\n" +
-            " If <b>Use Authentication</b> is disabled in previous dialog, <b>Edit Auth</b> is disabled here.</p>"
+            " must be unique.<br>After pressing <b>Edit Auth</b>, it's needed to " +
+            " Use <b>Authentication by Targets</b> and  <b>Authentication by Initiators</b> together. Then insert <b>UserID / Password</b> and <b>UserID_Mutual / Password_Mutual</b>." +
```

Comment:
Thanks for your review and comment. I think you are an expert in this area, because not everyone can understand "incoming" and "outgoing". Actually this fate intends to replace "incoming" and "outgoing" with better words, many users even in my team asked questions like "Does incoming at targets side mean outgoing at initiators side?". I think maybe "Authentication by targets" and "Authentication by initiators" are better, because it is easy to understand that "Authentication by Targets" means initiators should send user id and passwords to the targets, so that the targets can authenticated the initiators.

Thanks for your remind, I will also contact Document team add some explanations there